main.yml

```diff
@@ -0,0 +1,32 @@
+---
+- hosts: all
+  connection: local
+
+  vars_files:
+    - default.config.yml
+
+  roles:
+    - role: uninstalls
+      tags: ['uninstalls']
+    - role: command-line
+      tags: ['command-line']
+    - role: homebrew
+      tags: ['homebrew']
+    - role: rbenv
+      tags: ['rbenv']
+    - role: gems
+      tags: ['gems']
+    - role: postgresql
+      tags: ['postgresql']
+    - role: phpunit
+      tags: ['phpunit']
+    - role: wp-cli
+      tags: ['wp-cli']
+    - role: github
+      tags: ['github']
+    - role: signage-setup
+      tags: ['signage']
+    - role: blogs-setup
+      tags: ['blogs']
+    - role: inside-setup
+      tags: ['inside']
```

Comment:
I like the use of tags but I think they could be grouped together under a few tags. For example:

- `base`: `uninstalls`, `command-line`, `homebrew`
- `rails`: `rbenv`, `gems`, `postgresql`
- `php`: `phpunit`, `wp-cli`
- `apps`: `signage`, `blogs`, `inside`